scripts/zones/East_Ronfaure_[S]/mobs/Battrap.lua

```diff
@@ -0,0 +1,42 @@
+-----------------------------------
+-- Area: East Ronfaure [S]
+--  MOB: Battrap
+-- Note:  Place Holder for Goblintrap NM
+-- !pos 156 0 -438 81
+-----------------------------------
+
+require("scripts/zones/East_Ronfaure_[S]/MobIDs");
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+end;
+
+-----------------------------------
+-- onMobDeath
+-----------------------------------
+
+function onMobDeath(mob, player, isKiller)
+
+    local GOBLIN_TRAP_PH = GOBLIN_TRAP - 1;
+
+    if (mob:getID() == GOBLIN_TRAP_PH) then
+
+        local ToD = GetServerVariable("[POP]Goblintrap");
+
+        if (ToD <= os.time() and GetMobAction(GOBLIN_TRAP) == 0) then
```

Comment:
we stopped using get mob action.

also we started using an entirely different method for lotto pops where we don't even need a script for the NM anymore.
https://github.com/DarkstarProject/darkstar/blob/master/scripts/globals/mobs.lua#L54